### PR TITLE
feat: electrumx server setup

### PR DESCRIPTION
**Pull request by thephez**
Describe how to setup an ElectrumX server under a non-root user account

PR build of site viewable at https://dash-docs--78.org.readthedocs.build/en/78/network/electrumx-server.html

### CODE REVIEW

File: network/electrumx-server.rst [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+ 
+Set the server banner message and the coin type that the server will host:
+
+.. code-block:: shell
+
+  echo '$SERVER_SUBVERSION running on $DAEMON_SUBVERSION' > /home/electrumx/electrumx/banner
+  echo "/home/electrumx/electrumx/banner" > scripts/electrumx/env/BANNER_FILE
+  echo "Dash" > scripts/electrumx/env/COIN
+
+Set network values
```
Comment (strophy) [inline network/electrumx-server.rst:142]: Is this a heading? Should be underlined.

File: network/electrumx-server.rst [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+  echo "/home/electrumx/server.crt" > scripts/electrumx/env/SSL_CERTFILE
+  echo "/home/electrumx/server.key" > scripts/electrumx/env/SSL_KEYFILE
+
+Set the ElectrumX username to match our current user:
+
+.. code-block:: shell  
+
+  echo "electrumx" > scripts/electrumx/env/USERNAME
+
+Set the ElectrumX log directory
```
Comment (strophy) [inline network/electrumx-server.rst:195]: Is this a heading? Should be underlined or followed by a colon?

**Review by strophy — APPROVED** [review]
LGTM, utACK :+1:

File: network/electrumx-server.rst (hunk 1 above)
Comment (thephez) [reply network/electrumx-server.rst:142]: I couldn't decide and forgot to do either. Both seem a little weird, but I'll add a colon to be consistent with the preceding ones

**Pull request merged**